File: week1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>
int main(void)
{
    float dollars;
    int cents;
    int counter = 0;

    do 
    {
        dollars = get_float("Change owed: ");
    } 
    while (dollars < 0);

    cents = round(dollars * 100);

    while (cents >= 25)
    {
        cents = cents - 25;
        counter++;
    }

    while (cents >= 10)
    {
        cents = cents - 10;
        counter++;
    }

    while (cents >= 5)
    {
        cents = cents - 5;
        counter++;
    }

    while (cents >= 1)
    {
        cents = cents - 1;
        counter++;
    }
    printf("%d\n", counter);
}

File: pset6/cash/cash.py
from cs50 import get_float

while True:
    n = get_float("Change owed: ")
    if n > 0:
        num25 = (n * 100) // 25
        remainMoney = (n * 100) % 25
        print(f"num1: {num25}, {remainMoney}")
        num10 = remainMoney // 10
        remainMoney = remainMoney % 10
        print(f"num2: {num10}, {remainMoney}")
        num5 = remainMoney // 5
        remainMoney = remainMoney % 5
        print(f"num3: {num5}, {remainMoney}")
        num1 = remainMoney // 1
        remainMoney = remainMoney % 1
        print(f"num4: {num1}, {remainMoney}")
        print(f"{int(num25 + num10 + num5 + num1)}")
        break